lib/web/JsonpMainTemplate.runtime.js

```diff
@@ -16,7 +16,7 @@ module.exports = function() {
 		var script = document.createElement("script");
 		script.charset = "utf-8";
 		script.src = $require$.p + $hotChunkFilename$;
-		$crossOriginLoading$;
+		script.crossOrigin = $crossOriginLoading$;
```

Comment:
This will increase bundle size. Add a `if($crossOriginLoading$)` in front of this. The minimizer will handle this.

Reply:
@sokra Good call, fixed!